src/coreclr/vm/interoplibinterface_shared.cpp

```diff
@@ -31,6 +31,9 @@ void Interop::OnGCStarted(_In_ int nCondemnedGeneration)
 #ifdef FEATURE_COMWRAPPERS
         ComWrappersNative::OnBackgroundGCStarted();
```

Comment:
I think the name is a bit misleading. This is a Gen2 GC, not necessarily background though. 
Perhaps `OnGCStarted` is better?

Reply:
This is a legacy name - circa 2009 or so? I continued the pattern without putting much thought into it because I knew what it was implying. I am open to hearing better names from the experts.